charts/nextcloud/templates/cronjob.yaml

```diff
@@ -0,0 +1,136 @@
+---
+{{- if and .Values.cronjob.enabled .Values.cronjob.useCronJobInsteadOfCrond }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "nextcloud.fullname" . }}-cron
+  labels:
+    {{- include "nextcloud.labels"  ( dict "component" "cron" "rootContext" $ ) | nindent 4 }}
+    {{- with .Values.cronjob.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  {{- with .Values.cronjob.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  schedule: "*/5 * * * *"
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    metadata:
+      labels:
+        {{- include "nextcloud.selectorLabels"  ( dict "component" "cron" "rootContext" $ ) | nindent 8 }}
+        {{- with .Values.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: {{ .Chart.Name }}-cron
+              image: {{ include "nextcloud.image" . }}
+              imagePullPolicy: {{ .Values.image.pullPolicy }}
+              command:
```

Comment:
I suggest to change the command to:
```
runuser -u www-data -- /usr/local/bin/php -d php.memory_limit=2G cron.php
```
1. set memory limit and add option to tune it
2. use runuser to run it as www-data, otherwise files will change to root ownership in some cases.